Fabric.Terminology.Domain/Persistence/Mapping/IModelMapper.cs

```diff
@@ -2,10 +2,10 @@
 {
     using JetBrains.Annotations;
 
-    public interface IModelMapper<in TSrc, out TResult>
-        where TSrc : class
+    public interface IModelMapper<in TDto, out TResult>
+        where TDto : class, new()
```

Comment:
I thought that might be the case, but I couldn't see where you're using Activator.